test/projective/affine.jl

```diff
@@ -76,18 +76,34 @@ include("../imports.jl")
             @test_nowarn apply(tfm, keypoints)
             timage = apply(tfm, image)
             tkeypoints = apply(tfm, keypoints)
+            @test !any(isnan.(timage |> itemdata))
             @test length.(getbounds(timage).rs) == (25, 25)
```

Comment:
Would you mind adding checks to the full range range in these tests, not just the size. Down the road, I can image the ranges becoming incorrect again without a test
```suggestion
            @test getbounds(timage).rs == (1:25, 1:25)
```